src/toxcore/net_crypto/mod.rs

```diff
@@ -173,31 +173,6 @@ impl NetCrypto {
         self.connections.read().get(&pk).cloned()
     }
 
-    /// Create `CookieResponse` packet with `Cookie` requested by `CookieRequest` packet
-    fn handle_cookie_request(&self, packet: &CookieRequest) -> Result<CookieResponse, Error> {
```

Comment:
Why did you move this function from NetCrypto into DhtServer? That is not a solution.